terraform/main.tf

```diff
@@ -0,0 +1,104 @@
+terraform {
+  required_version = ">= 0.12"
+
+  backend "remote" {
+    hostname     = "app.terraform.io"
+    organization = "geekzone"    
+
+    workspaces {
+      name = "cicd-ec2"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-2"
+}
+
+
+module "networking" {
+  source = "./networking"
+  cidr   = "10.0.0.0/16"
+
+  az-subnet-mapping = [
+    {
+      name = "subnet1"
+      az   = "eu-west-2a"
+      cidr = "10.0.0.0/24"
+    },
+    {
+      name = "subnet2"
+      az   = "eu-west-2c"
+      cidr = "10.0.1.0/24"
+    },
+  ]
+}
+
+# Create a security group 
+resource "aws_security_group" "allow-ssh-and-egress" {
+  name = "main"
+
+  description = "Allows SSH traffic into instances as well as all eggress."
```

Comment:
typo at the end of the description: eggress

Reply:
well spotted.